conformance/src/test/scala/za/co/absa/enceladus/conformance/interpreter/InterpreterSuite.scala

```diff
@@ -42,7 +42,7 @@ class InterpreterSuite extends FunSuite with SparkTestBase with BeforeAndAfterAl
   }
 
   def testEndToEndDynamicConformance(useExperimentalMappingRule: Boolean): Unit = {
-    // Enable Conformance Framweork
+    // Enable Conformance framework
```

Comment:
```suggestion
    // Enable Conformance Framework
```
You've kept the capitalization elsewhere, let's keep it here as well.